Superlista/Views/AddCollaboratorSheetView.swift

```diff
@@ -47,7 +47,7 @@ struct AddCollaboratorSheetView: View {
 
                                 Button(action: {
                                     self.showShareActionSheet.toggle()
-                                    shareSheet(listID: list.id, option: "1", listName: list.title, ownerName: listOwner.name!)
+                                    shareSheet(listID: list.id, option: "1", listName: list.title, ownerName: listOwner.name ?? "")
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()